web/src/main/java/se/inera/intyg/webcert/web/web/controller/authtestability/UserResource.java

```diff
@@ -144,7 +144,18 @@ public Response setSjf() {
     public Response setRef(@PathParam("refValue") String refValue) {
         webCertUserService.getUser()
             .setParameters(
-                new IntegrationParameters(refValue, null, null, null, null, null, null, null, null, true, false, false, true));
+                new IntegrationParameters(refValue, null, null, null, null, null, null, null, null, true, false, false, true, null));
+        return Response.ok().build();
+    }
+
+    @POST
+    @Path("/parameters/launchId")
+    @PrometheusTimeMethod
+    public Response setLaunchId() {
```

Comment:
Maybe add launchId as a path-param so the user of the testability can decide what to set.